tools/wake-format/formatter.h

```diff
@@ -174,7 +174,7 @@ struct NestAction {
   NestAction(FMT formatter) : formatter(formatter) {}
 
   ALWAYS_INLINE void run(wcl::doc_builder& builder, ctx_t ctx, CSTElement& node) {
-    builder.append(formatter.compose(ctx.nest(), node));
+    builder.append(formatter.compose(ctx.sub(builder).nest(), node));
```

Comment:
Oh shit right lol...whoopsy

The rule stated was that if you throw out the builder you have to sub....I somehow turned a blind eye to the nest case lol.

I think if `nest` had the affect of also adding a newline we would have been right...oh well.